### PR TITLE
Add installation for Windows and change download deployer via console for Linux/MacOS

### PR DESCRIPTION
**Pull request by andreybolonin**

### CODE REVIEW

**Comment by elfet** [comment]
@andreybolonin please rebase and fix conflicts!

**Comment by andreybolonin** [comment]
All is ok?

**Comment by oanhnn** [comment]
Why don't you merge multiple commits to one?
http://www.git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages

File: getting-started.md [hunk 1]
```diff
@@ -100,8 +102,8 @@ task('reload:php-fpm', function () {
 after('deploy', 'reload:php-fpm');
 ```
 
-> If you want to see exactly what deployer does on you server, then just run it in a more verbose mode: `$ dep deploy -vvv`.
+> If you want to see exactly what deployer does on you server, then just run it in a more verbose mode: `dep deploy -vvv`.
 
-Finally: configure your server to serve your public directory inside `current`.
+Finally: configure you server to serve your public directory inside `current`.
```
Comment (elfet) [inline getting-started.md]: May be `your`?

**Comment by antonmedv** [comment]
Please rebase.

**Pull request closed without merging**